### PR TITLE
Rodriguezd 1149676 ga membership api

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Created new branch rodriguezd-1149676GAmembershipAPI
Updated the Methods table with the following methods in the resource type file:
-	List members
-	Add members
-	Remove members
-	Change member’s role
-	Get team photo
Removed the following methods from the v1.0 of team.md
-	Publish apps to your organization
-	Add app to team
-	Add tab to channel

### CODE REVIEW

File: api-reference/v1.0/api/team-get-photo.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+---
```
Comment (nkramer) [inline api-reference/v1.0/api/team-get-photo.md:1]: Please remove this file, teen photo is not part of team membership and is not in v1.0

**Review by laujan — APPROVED** [review]

**Review by Lauragra — CHANGES_REQUESTED** [review]
This PR requires an update to the changelog and the TOC as well. Thanks.

File: api-reference/v1.0/toc.yml [hunk 2]
```diff
@@ -4830,6 +4830,16 @@ items:
           href: api/team-get.md
         - name: Update team
           href: api/team-update.md
+        - name: Delete team
```
Comment (Lauragra) [inline api-reference/v1.0/toc.yml:4833]: For some reason the TOC links don't appear to be working, I'm not sure why. I can't verify the TOC entries on the review site so I'll hold off on publishing this for now.

Comment (laujan) [reply api-reference/v1.0/toc.yml:4833]: Hi @Lauragra I updated the delete team path and now all of the links seems to be working.

**Review by laujan — APPROVED** [review]

**Review by laujan — APPROVED** [review]

**Comment by laujan** [comment]
Hi @Lauragra: I think this is ready-to-merge.

**Pull request merged**